common/logger/src/logger.rs

```diff
@@ -44,6 +45,11 @@ pub fn set_global_logger(logger: Arc<dyn Logger>) {
     if LOGGER.set(logger).is_err() {
         eprintln!("Global logger has already been set");
     }
+    tracing::subscriber::set_global_default(
+        crate::tracing_adapter::TracingToDiemLoggerLayer
+            .with_subscriber(tracing_subscriber::Registry::default()),
+    )
+    .unwrap();
```

Comment:
The current semantics of this function is that trying to set the global logger twice will not panic and crash the node, unwrapping this breaks those semantics. Can you change this to not panic?